src/uct/ib/configure.m4

```diff
@@ -453,3 +454,7 @@ m4_include([src/uct/ib/rdmacm/configure.m4])
 AC_DEFINE_UNQUOTED([uct_ib_MODULES], ["${uct_ib_modules}"], [IB loadable modules])
 AC_CONFIG_FILES([src/uct/ib/Makefile
                  src/uct/ib/ucx-ib.pc])
+AS_IF([test -n "$have_devx" -a \( "x$with_mlx5_dv" = xyes -o "x$with_mlx5_hw" = xyes \) ],
```

Comment:
we should not have set have_devx to yes when with_mlx5_dv is not yes
IMO it should have been 
`AM_CONDITIONAL([HAVE_DEVX], [test "x$have_devx" = xyes])`

Reply:
there are 3 variables tested independently, merging it into single flow will make code really complex: have to add ```have_devx=no``` code into all branches where with_mlx5_dv failed to detect or skipped

Reply:
ok, but not clear why with_mlx5_hw is needed?

Reply:
mlx5_hw comes from exp verbs, and does not include devx